### PR TITLE
Remove Uh oh toast notification

### PR DESCRIPTION
**Pull request by sujitacharya2005**
Resolves https://github.com/brave/brave-browser/issues/28514

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java [hunk 1]
```diff
@@ -1084,11 +1072,6 @@ public void onBalance(int errorCode) {
                 TextView usdBalanceText = mPopupView.findViewById(R.id.usd_balance_text);
                 usdBalanceText.setText(usdText);
             }
-        } else if (errorCode == BraveRewardsNativeWorker.LEDGER_ERROR) { // No Internet connection
-            String args[] = {};
-            Log.e(TAG, "Failed to fetch rewards parameters from server");
-            showNotification(
-                    REWARDS_NOTIFICATION_NO_INTERNET_ID, REWARDS_NOTIFICATION_NO_INTERNET, 0, args);
```
Comment (deeppandya) [inline android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java:1091]: we may need to show different notification for LEDGER_ERROR. CC @Miyayes

File: android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java [hunk 2]
```diff
@@ -735,18 +735,6 @@ private void showNotification(String id, int type, long timestamp, String[] args
                         break;
                 }
                 break;
-            case REWARDS_NOTIFICATION_NO_INTERNET:
```
Comment (deeppandya) [inline android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java:738]: @Miyayes should we show different kind of notification when internet is not available and user tries to open rewards panel ?

File: android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java (hunk 1 above)
Comment (Miyayes) [reply android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java:1091]: @deeppandya I don't think we show anything on desktop, so we don't need to show anything here?

File: android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java (hunk 2 above)
Comment (Miyayes) [reply android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java:738]: @emerick Do we show anything on desktop? Let's match desktop. My impression is that this "Uh oh" notification just doesn't exist anymore in general on desktop, so wanted to match.

Comment (emerick) [reply android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java:738]: No, we don't do/show anything specific in that scenario on desktop.

File: android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java (hunk 1 above)
Comment (emerick) [reply android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java:1091]: Yeah, it doesn't seem necessary to show anything here (desktop doesn't, from what I can see).

File: android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java (hunk 2 above)
Comment (Miyayes) [reply android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java:738]: @deeppandya Okay, then good to remove without replacement. Thanks.

File: android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java (hunk 1 above)
Comment (Miyayes) [reply android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java:1091]: @deeppandya Okay, then good to remove without replacement. Thanks.

File: android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java (hunk 2 above)
Comment (deeppandya) [reply android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java:738]: Sounds good @Miyayes .

**Review by deeppandya — APPROVED** [review]
LGTM

**Pull request merged**